_posts/2018-01-02-this-is-not-a-test.md

```diff
@@ -0,0 +1,38 @@
+---
+layout: post
+title:  "This Is Not a Test"
+date:   2018-01-02 12:00:00 -0400
+categories: culture interviews
+author: Janice Smith
+---
+
+At Forward Financing, we are always trying to improve how we work.
+We have some core tenets to how we work as a team. Our director of engineering,
+Patrick Hereford, wrote a great post about that recently. You should check it out [here](http://tech.forwardfinancing.com/engineering/culture/2017/09/30/our-engineering-culture-methodologies-and-philosophies.html){:target='_blank'}
+
+We have been in the middle of growing the engineering team numbers here,
+in a push that will eventually more than double the size of our group.
+This growth is key to support all of the new work coming our way,
```

Comment:
I know I said replace "which" with "that", but I think changing this from "that marks" to "marking" reads better.